.gitignore

```diff
@@ -106,3 +106,9 @@ CMakeSettings.json
 
 # Python pickle files
 *.pkl
+
+# Large fules from partitioner builds
+*.omcp
+# Building scripts in root folder
+debug-*.sh
```

Comment:
I think only you use this, so might want to remove it.

Reply:
Removed.